sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -870,7 +870,13 @@ public long count(CaseCriteria caseCriteria) {
 						pseudonymizer.pseudonymizeDto(SymptomsDto.class, visitSymptoms, inJurisdiction, null);
 
 						exportDto.setLastCooperativeVisitSymptoms(SymptomsHelper.buildSymptomsHumanString(visitSymptoms, true, userLanguage));
-						exportDto.setLastCooperativeVisitSymptomatic(visitSymptoms.getSymptomatic() ? YesNoUnknown.YES : YesNoUnknown.NO);
+						if (visitSymptoms.getSymptomatic() == YesNoUnknown.YES) {
```

Comment:
Why not just use `exportDto.setLastCooperativeVisitSymptomatic(visitSymptoms.getSymptomatic())`?